src/test/java/com/example/datavisualizationbackend/ApplicationTests.java

```diff
@@ -17,21 +17,21 @@
 @RunWith(SpringJUnit4ClassRunner.class)
 public class ApplicationTests {
 
-	//Test all servlet related calls. Can test endpoints by calling them directly using MockMvc.
-	private MockMvc mockMvc;
+    //Test all servlet related calls. Can test endpoints by calling them directly using MockMvc.
+    private MockMvc mockMvc;
 
-	@InjectMocks
-	private TimeController timeController;
+    @InjectMocks
+    private UploadEventController uploadEventController;
 
-	@Before
-	public void setUp() throws Exception{
-		mockMvc = MockMvcBuilders.standaloneSetup(timeController).build();
-	}
+    @Before
+    public void setUp() throws Exception {
+        mockMvc = MockMvcBuilders.standaloneSetup(uploadEventController).build();
+    }
 
-	@Test
-	public void testTimeController() throws Exception{
-		mockMvc.perform(get("/")
-				.accept(MediaType.APPLICATION_JSON))
-				.andExpect(status().isOk());
-	}
+    @Test
+    public void testUploadEventController() throws Exception {
```

Comment:
Prefixing all tests methods with test is a throwback to old JUnit where that was how it worked out what was a test.

Now that it uses the Test annotation it's less noise if you exclude it e.g.

```
@Test
public void getUploadEventsReturnsOKStatus() throws Exception
```